angular-selector.md

```diff
@@ -0,0 +1,76 @@
+# Angular Selector Extentions
+
+## Preparation
+Ensure that your application bootstrapped in development mode.
+By default, this is true, unless code contains [enableProdMode](https://angular.io/api/core/enableProdMode) method call.
+
+If your application run in production mode then `AngularSelector` will not work.
+
+## Usage
+
+### WaitForAngular
+To wait until Angular'component tree was loaded need to add `waitForAngular` method into fixture's `beforeEach` hook.
+
+```js
+import { waitForAngular } from 'testcafe-angular-selectors';
+
+fixture `App tests`
+    .page('http://angular-app-url')
+    .beforeEach(async t => {
+        await t.expect(waitForAngular()).ok();
+    });
+```
+Sometime Angular's component tree is very large or your application has a large startup time.
+In this case the assertion `await t.expect(waitForAngular()).ok();` may failed.
+To prevent this you can increase timeout for assertion.
```

Comment:
```
Sometimes the Angular's component tree is very large or your application takes a lot of time to start up.
In this case, the assertion `await t.expect(waitForAngular()).ok();` may fail.
To prevent this, you can increase timeout for the assertion.
```